cl/search/models.py

```diff
@@ -58,6 +58,8 @@
     ("ZR", "columbia merged with resource.org"),
     ("ZCR", "columbia merged with court and resource.org"),
     ("ZL", "columbia merged with lawbox"),
+    ("U", "Harvard, Library Innovation Lab Case Law Access Project"),
+    ("CU", "court website merged with Harvard"),
```

Comment:
The dumb thing about this scheme is that we're going to need all the other "Combined with" possibilities. This should eventually be a m2m field, but until we can do that, we're stuck with this dumb (old) system that I invented. We can do all of those other possibilities now or later....

Reply:
Either way works for me.